kombu/transport/redis_cluster.py

```diff
@@ -27,8 +28,15 @@
 logger = get_logger(__name__)
 
 
-# Override this method to use other redis client
-def create_redis_cluster_connection(hostname, port, password, ssl):
+# Override these methods to use other redis client
+def create_redis_cluster_connection_for_consumer(hostname, port, password, ssl):
```

Comment:
It seems to that it is a duplicated function. May I ask a reason?

Reply:
We want to maintain separate redis cluster pool for producer and consumer, so these methods creates producer and consumer redis cluster clients. It has same logic for default implementation, but they are meant to be overridden by app to make it able to utilize different redis cluster clients.

Anyway, I'll update code to remove duplicates as duplicates can be eliminated.

Reply:
Oh I see. It's for allowing to use different clients for consumer and producer.
Thanks for the explanation.